django_bitcoin/tasks.py

```diff
@@ -41,74 +42,78 @@ def NonBlockingCacheLock(key, lock=None, blocking=False, timeout=10000):
 @task()
 def query_transactions():
     logger = get_task_logger('bitcoin_transactions')
+    logger.info("query_transactions: start task")
 
     with NonBlockingCacheLock("query_transactions_ongoing"):
         try:
             info = bitcoind.bitcoind_api.getinfo()
+            testnet = info['testnet']
+
+            blockcount = bitcoind.bitcoind_api.getblockcount()
```

Comment:
Are `bitcoind` calls 100% necessary on every task execution?

This will slow it drammatically